core/src/main/java/org/testcontainers/containers/VncRecordingSidekickContainer.java

```diff
@@ -1,94 +0,0 @@
-package org.testcontainers.containers;
```

Comment:
is this an unrelated change?

Reply:
there are a lot of changes coming from @bsideup's pr